benchmarks/free_surface_tractions/viscous/free_surface_viscous_cylinder_3D_loading.prm

```diff
@@ -0,0 +1,162 @@
+
+  subsection Box
+    set X repetitions = 10
+    set Y repetitions = 10
+    set Z repetitions = 10
+    set X extent      = 500e3
+    set Y extent      = 500e3
+    set Z extent      = 500e3
+  end
+end
+
+# Mesh refinement specifications 
+subsection Mesh refinement
+  set Initial adaptive refinement        = 1
+  set Initial global refinement          = 1
+  set Time steps between mesh refinement = 1
+  set Strategy				 = strain rate
+end
+
+# Element types
+subsection Discretization
+  set Composition polynomial degree     = 2
+  set Stokes velocity polynomial degree = 2
+  set Temperature polynomial degree     = 1
+end
+
+subsection Free surface
+  set Free surface boundary indicators = top
+  set Surface velocity projection      = normal
+  set Additional tangential mesh velocity boundary indicators = left,right,front,back
+end
+
+# Velocity boundary conditions
+subsection Boundary velocity model
+  set Tangential velocity boundary indicators = bottom, left, right, front, back
+end
+
+# Prescribe a fixed vertical traction on the top boundary
+subsection Boundary traction model
+  set Prescribed traction boundary indicators = top z: function
+end
+
+subsection Boundary traction model
+  subsection Function
+    set Variable names = x,y,z,t
+    set Function constants  = r0=100.e3, H0=1.e3, t1=1.e3, rhoi=900, g=9.8, t0=1.e3
+        # r0 is load radius, H0 is load height, t1 is time load is removed, 
+	# rhoi is density of ice/load, g is gravitational acceleration 
+    set Function expression = 0; 0; if ((x^2+y^2)<r0^2 && t<t1,-g*H0*rhoi, 0)
+  end
+end
+
+# Number and name of compositional fields
+subsection Compositional fields
+  set Number of fields = 6
+  set Names of fields  = stress_xx, stress_yy, stress_zz, stress_xy, stress_xz, stress_yz
+end
+
+# Spatial domain of different compositional fields
+subsection Initial composition model
+  set Model name = function
+  subsection Function
+    set Variable names      = x,y,z
+    set Function constants  = 
+    set Function expression = 0; 0; 0; 0; 0; 0;
+  end
+end
+
+# Composition boundary conditions
+subsection Boundary composition model
+  set Allow fixed composition on outflow boundaries = true
+  set Fixed composition boundary indicators   = 
+  set List of model names = initial composition
+end
+
+# Temperature boundary conditions
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators = bottom, top, left, right, front, back
+  set List of model names = box
+  subsection Box
+    set Bottom temperature = 293
+    set Left temperature   = 293
+    set Right temperature  = 293
+    set Top temperature    = 293
+    set Front temperature  = 293
+    set Back temperature   = 293
+  end
+end
+
+# Temperature initial conditions
+subsection Initial temperature model
+  set Model name = function
+  subsection Function
+    set Function expression = 293
+  end
+end
+
+# Material model
+subsection Material model
+
+   set Model name = simple
```

Comment:
For simplicity, you could use the `viscoelastic` material model here and set the `Elastic shear moduli` to something like 1e50, which for all intensive purposes produces viscous behavior (e.g., negligible viscoelastic contribution).